packages/pluggableWidgets/datagrid-web/src/Datagrid.editorConfig.ts

```diff
@@ -139,6 +139,7 @@ export const getPreview = (values: DatagridPreviewProps): StructurePreviewProps
                     type: "Container",
                     borders: true,
                     grow: column.width === "manual" && column.size ? column.size : 1,
+                    backgroundColor: values.columnsHidable && column.hidable === "hidden" ? "#F5F5F5" : undefined,
```

Comment:
Nit: Extract `values.columnsHidable && column.hidable` into a const or function in case the naming ever changes in the future.

Reply:
tbh, if name changes typescript will be in place to complain, also if i extract it will add an extra {} and a const to be used in just one place. :)